requirements.txt

```diff
@@ -0,0 +1,5 @@
+scipy==0.18.1
+torch==0.1
```

Comment:
Pytorch cannot currently be installed directly from pypi. I would suggest removing this and adding a note in the readme pointing the user to install the proper version (via their requirements for cuda etc) from here: http://pytorch.org/